python/gps/algorithm/policy/policy_prior.py

```diff
@@ -30,3 +30,44 @@ def eval(self, Ts, Ps):
                        prior_fd.dot(sig).dot(prior_fd.T) + prior_cond])
         ])
         return np.zeros(dX+dU), Phi, 0, self._hyperparams['strength']
+
+    def fit(self, X, pol_mu, pol_sig):
```

Comment:
It's no different, but we need a fit function here in case someone uses `PolicyPrior` and not `PolicyPriorGMM`. There's only difference between `dynamics_lr.fit` and `dynamics_lr_prior.fit` because the former doesn't have a real `eval` function, so it can't use `gauss_fit_joint_prior` and needs to have the method explicitly written out. This could definitely be improved, but I figured it was fine for now.